eng/SharedFramework.External.props

```diff
@@ -114,6 +114,7 @@
     <_CompilationOnlyReference Include="System.ComponentModel.Annotations" />
     <_CompilationOnlyReference Include="System.Runtime.CompilerServices.Unsafe" />
     <_CompilationOnlyReference Include="System.Text.Json" />
+    <_CompilationOnlyReference Include="System.Text.Encodings.Web" />
```

Comment:
@wtgodbe due to package version conflicts between App.Ref and other projects, I suspect this is needed both here and below. Do you remember if that's correct❔